AutoGr/settings-sample.py

```diff
@@ -126,10 +127,6 @@
 LOGIN_REDIRECT_URL = 'home'
 
 RUN_API_URL = "http://127.0.0.1:8000/autograde/api/"
-EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
`EMAIL_BACKEND` removed?

Reply:
It was a duplicate entry. `EMAIL_BACKEND` is also defined on line 200.

Reply:
Hnn, ok. Please go ahead and merge. 

Thanks.

Reply:
`AUTHENTICATION_BACKENDS` is the actual thing which is added in settings.py file (in the last line) but for some reason it isn't shown here.

https://github.com/BilalZaib/AutoGrader/blob/3dc4390591ea147696ba172bd04ea6ec8a49fb32/AutoGr/settings-sample.py

Reply:
This allows inactive user to login.

Reply:
It shows in the commit (just not in this comment page).

Reply:
Okay 👍